modules/ROOT/pages/configure-client-management-azure.adoc

```diff
@@ -59,7 +59,7 @@ For a list of supported grant types, see the https://docs.microsoft.com/en-us/az
 
 === Client Secrets
 
-Initial secrets expiration is set to 6 months per Microsoft recommendation.
+Initial secrets expiration is set to 6 months per Microsoft's recommendation. You can reset client secrets via Anypoint Platform. Anypoint Platform creates new secrets on your behalf but does not support removing the old secrets from your Microsoft account on your behalf. However, you can still remove the old secrets in Microsoft accounts.
```

Comment:
where can customers find step by step information to reset secrets? should we add that here?

Reply:
Reset client secret - https://docs.mulesoft.com/api-experience-hub/my-applications#reset-client-secret